authors/apps/authentication/tests/test_social_auth.py

```diff
@@ -1,35 +1,38 @@
 from django.test import TestCase
+from django.urls import reverse
 from rest_framework import test, status
 
 
 class SocialAuthTest(TestCase):
+    def setUp(self):
+        self.client = test.APIClient()
 
     # FACEBOOK
-    def test_user_signup_success(self):
-        pass
-
-    def test_user_signup_invalid_fields(self):
-        pass
-
-    def test_empty_fields(self):
-        pass
-
-    # TWITTER
-    def test_user_signup_success(self):
-        pass
-
-    def test_user_signup_invalid_fields(self):
-        pass
-
-    def test_empty_fields(self):
-        pass
+    def test_user_signup_page_redirects_to_facebook(self):
+        """
+            Ensures that the redirects are found, basically tests
+            that the configurations are functional
+        """
+        res = self.client.get(
+            reverse('social:begin', kwargs={'backend': 'facebook'}))
+        self.assertEqual(res.status_code, status.HTTP_302_FOUND) 
+
+    # Github
+    def test_user_signup_page_redirects_to_github(self):
+        """
+            Ensures that the redirects are found, basically tests
+            that the configurations are functional
+        """
+        res = self.client.get(
+            reverse('social:begin', kwargs={'backend': 'github'}))
+        self.assertEqual(res.status_code, status.HTTP_302_FOUND)
 
     # GOOGLE
-    def test_user_signup_success(self):
-        pass
-
-    def test_user_signup_invalid_fields(self):
-        pass
-
-    def test_empty_fields(self):
-        pass
+    def test_user_signup_page_redirects_to_github(self):
```

Comment:
I think the test name should be `test_user_signup_page_redirects_to_google` instead of `test_user_signup_page_redirects_to_github`